choose-thumbnail/index.md

```diff
@@ -0,0 +1,19 @@
+## Choosing a thumbnail image
+
+Think of your game's thumbnail as its box cover if it was sold off a shelf in a store (they used to do that, you know). It should attract attention and give an impression of what the game is like.
+
+A good thumbnail should be more than just a title floating on a blank background. Many good thumbnails are simply well-framed screenshots with the game's title added. Other good ones are images created specifically for the thumbnail using art from the game.
+
+Don't stress out too much about getting your image to the correct size; there's a handy cropper tool to make it easy for you. But you should still start out with a high-quality original image.
+
+Only use images that you created. Don't use another game's image, an internet meme, something you found on Deviantart, or anything like that.
+
+### Mature content in thumbnails
+
+Please don't choose an image for your thumbnail that contains nudity, swear words, or adult-oriented imagery. This only applies to thumbnails, not to other images, because they show up on public lists and in search results; once someone visits a game page, there is a maturity rating on display and someone can make their own decision about whether to stay on the page and play the game.
```

Comment:
suggestion: change "because they show up on public lists and in search results;" to "because thumbnails show up on public lists and in search results;"